rcon/hooks.py

```diff
@@ -184,6 +185,13 @@ def ban_if_has_vac_bans(rcon: RecordedRcon, steam_id_64, name):
             except:
                 logger.exception("Unable to send vac ban to audit log")
 
+def kick_if_low_playtime(rcon, struct_log, steam_id_64):    
+    hll_timm_played = get_hll_playtime_in_hours(steam_id_64)
+    if hll_timm_played is None or hll_timm_played < MIN_HLL_PLAYTIME_HOURS:
```

Comment:
So that also kicks players that have a private profile?

For other features like this I add an immunity flag and I also make the VIPs safe example here: https://github.com/MarechJ/hll_rcon_tool/blob/master/rcon/auto_kick.py#L30  and there https://github.com/MarechJ/hll_rcon_tool/blob/master/rcon/game_logs.py#L377